backend/Application/Api/GraphQL/Import/AccountChangeCalculator.cs

```diff
@@ -152,18 +151,15 @@ public AccountReleaseScheduleItem[] GetAccountReleaseScheduleItems(IEnumerable<T
                 .Distinct();
             var accountIdMap = _accountLookup.GetAccountIdsFromBaseAddresses(distinctBaseAddresses);
 
-            toInsert = result
-            .Where(x => accountIdMap.ContainsKey(x.AccountBaseAddress) && accountIdMap[x.AccountBaseAddress].HasValue)
-            .Where(x => accountIdMap.ContainsKey(x.FromAccountBaseAddress) && accountIdMap[x.FromAccountBaseAddress].HasValue)
-            .Select(x => new AccountReleaseScheduleItem
-            {
-                AccountId = accountIdMap[x.AccountBaseAddress] ?? throw new InvalidOperationException("Account does not exist!"),
-                TransactionId = x.TransactionId,
-                Index = x.ScheduleIndex,
-                Timestamp = x.Timestamp,
-                Amount = x.Amount,
-                FromAccountId = accountIdMap[x.FromAccountBaseAddress] ?? throw new InvalidOperationException("Account does not exist!"),
-            })
+            toInsert = result.Select(x => new AccountReleaseScheduleItem
+                {
+                    AccountId = accountIdMap[x.AccountBaseAddress] ?? throw new InvalidOperationException("Account does not exist!"),
+                    TransactionId = x.TransactionId,
+                    Index = x.ScheduleIndex,
+                    Timestamp = x.Timestamp,
+                    Amount = x.Amount,
+                    FromAccountId = accountIdMap[x.FromAccountBaseAddress] ?? throw new InvalidOperationException("Account does not exist!"),
+                })
```

Comment:
This is just reverting the workaround where some accounts were missing, right?

Reply:
yes